projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.html

```diff
@@ -0,0 +1,119 @@
+<ng-container *ngIf="messageEvents$ | async as messageEvents">
```

Comment:
@morganm58 what do you think of the UI? I believe the accessibility was covered by @developpeurweb
Tbh it looks good to me. 

How to test and get to this page?

1. go to this branch
2. enable b2b by doing `yarn start:b2b`
3. login as mark rivers (mark.rivers@rustic-hw.com) or any other b2b users
4. visit http://localhost:4200/powertools-spa/en/USD/my-account/support-ticket/00000001 


--------
Not relevant to this comment above, but the data flow for this 'generic' component. I'll need to get approval for. Will pull another developer or bring in catchup for Wednesday, September 21st because the more I look at it, it seems to me like a specific component just for customer ticketing. 

One can say, we will re-use this styles across different other components/features, but maybe not? Moreover, if ever the data structure that comes from the backend for 'other' messaging component is different then they'd have to serialize it to conform to our data, which is fine, but please hold on for a second.